src/jobs/release-lerna-packages.yml

```diff
@@ -122,3 +130,27 @@ steps:
       condition:
         not: <<parameters.dryrun>>
       steps: <<parameters.post-job-steps>>
+
+  # clean up the CTC cases
+  - when:
+      condition:
+        equal: ['latest', <<parameters.tag>>]
+      steps:
+        - run:
+            when: on_fail
+            name: Close CTC case as Not Implemented
+            command: |
+              if [ -z "${SF_CHANGE_CASE_SFDX_AUTH_URL}" ] || [ -z "${SF_CHANGE_CASE_TEMPLATE_ID}" ] || [ -z "${SF_CHANGE_CASE_SCHEDULE_BUILD}" ]; then
+                  echo "Environment not configured for CTC"
+              else
+                  sfchangecase close --location "https://github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME" --status "Not Implemented"
```

Comment:
This is interesting. The `CIRCLE_PROJECT_REPONAME` is the name of the lerna repo, which may or may not release several packages. We have no idea what packages were released based on the change case only. We will have the history in the github releases though. 

Not something we should address now, just pointing it out. If it ever becomes an issue, we could add the list of packages released on the case somewhere.

Reply:
yeah--we can't have multiple change cases open simultaneously on the same repo (location).